package.json

```diff
@@ -132,7 +133,9 @@
   },
   "dependencies": {
     "bluebird": "^3.5.1",
-    "clappr": "0.2.78",
+    "clappr": "^0.2.85",
```

Comment:
This is no longer on dev and should be removed from this PR. Same w/ `hh-mm-ss`. Please run 

`npm uninstall hh-mm-ss && npm install --save clappr@0.2.78` to ensure that `package-lock.json` is changed accordingly

Reply:
changing

Reply:
The upgrade did not work. Did you run `npm uninstall hh-mm-ss && npm install --save clappr@0.2.78`?

Reply:
Also remove `hh-mm-ss`